src/ecosystem/connectivity.jl

```diff
@@ -187,7 +188,7 @@ function connectivity_strength(TP_base::AbstractMatrix{Float64})::NamedTuple
     C2 = outdegree_centrality(g)
 
     # For each node, find strongly connected predecessor (by number of connections)
-    strong_pred = zeros(Int64, size(C1)...)
+    strong_pred = zeros(Int64, (size(C1)...))
```

Comment:
Did you add the parenthesis in or was this the formatter?

Reply:
I think this would work without the splat or parenthesis

```suggestion
    strong_pred = zeros(Int64, size(C1))
```